src/convertToObject.js

```diff
@@ -5,8 +5,23 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(styles) {
+  const styleObject = {};
+
+  const declarations = styles.split(';');
+
+  declarations.forEach((declaration) => {
+    const [key, value] = declaration.split(':');
+
+    if (key && value) {
```

Comment:
[CODE KNOWLEDGE] - Good use of a guard clause to ensure that only valid key-value pairs are added to the object. This prevents adding empty or malformed styles.